README.md

```diff
@@ -33,6 +33,12 @@ Gears could be used together or alone.
 - [![Version](https://img.shields.io/maven-central/v/com.redmadrobot.extensions/resources-ktx?style=flat-square&label=resources-ktx)][resources-ktx] - A set of extensions for accessing resources
 - [![Version](https://img.shields.io/maven-central/v/com.redmadrobot.extensions/viewbinding-ktx?style=flat-square&label=viewbinding-ktx)][viewbinding-ktx] - A set of extensions for dealing with ViewBinding
 
+### :mag_right: **[ViewModelEvents](viewmodelevents/)**
```

Comment:
Maybe better to use :speech_balloon: `:speech_balloon:` here?

Reply:
🤔

Reply:
I think we could, but would it be better to leave it for TextValue?

Reply:
Otherwise i don't have any ideas which emoji we should use there :D